src/views/mfa-verify/CallAndSMSForm.js

```diff
@@ -57,19 +57,19 @@ define(['okta', 'vendor/lib/q', 'views/shared/TextBox'], function (Okta, Q, Text
         className: 'button ' + buttonClassName,
         title: formSubmit,
         click: function () {
-          self.clearErrors();
+          form.clearErrors();
+          this.disable();
+          this.options.title = formSubmitted;
+          this.render();
           // To send an OTP to the device, make the same request but use
           // an empty passCode
           this.model.set('answer', '');
           this.model.save()
-          .then(_.bind(function () {
-            this.options.title = formSubmitted;
```

Comment:
I think changing the title to whatever formSubmitted is (call or sms) should still be done after save. Before save we haven't really submitted anything. So if there is an error the button will read "Sent" even though nothing was actually sent.
```clearErrors``` makes sense to be moved before saving the model.